diff --git build-aux/release-suffix build-aux/release-suffix
index cca1cd0..b268139 100755
--- build-aux/release-suffix
+++ build-aux/release-suffix
@@ -1,7 +1,7 @@
 #!/bin/sh

 # Is this a release - build from tag?
-if git describe --tags --match "v[0-9]*" | grep -q '-'; then
+#if git describe --tags --match "v[0-9]*" | grep -q '-'; then
     # This is a master build, we want to make every build
     # newer than all the previous builds using a timestamp,
     # and make it easy to locate the commit from the build
@@ -9,6 +9,6 @@ if git describe --tags --match "v[0-9]*" | grep -q '-'; then
     release_suffix=.$(date -u +%Y%m%d%H%M)
     commithash=$(git rev-parse --short HEAD)
     release_suffix+=.git$commithash
-fi
+#fi

-echo $release_suffix
\ No newline at end of file
+echo $release_suffix
